infra/docker/jupyter/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM jupyter/minimal-notebook:619e9cc2fc07
+
+USER root
+WORKDIR /feast
+
+# .git (required by SDK setup.py)
+COPY .git .git
```

Comment:
.git and README:md are only needed by " pip install -e sdk/python" IIRC, so if you move them down you can reuse layer with requirements installed. (.git folder changes with every commit)

Reply:
Corrected, thanks.